Task3/MoneyManager/MoneyManager.DataAccess/Repositories/AssetRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Linq.Expressions;
+using MoneyManager.DataAccess.Models;
+
+namespace MoneyManager.DataAccess.Repositories
+{
+    public class AssetRepository : BaseRepository<Asset>
+    {
+        public AssetRepository(IApplicationContext applicationContext) : base(applicationContext)
+        {
+
+        }
+
+        public IEnumerable<Asset> GetAllAssetsByUserIdOrderedByAssetName(int userId)
```

Comment:
Of course, it depends on a project, but usually ordering isn't included in the method's name.